packages/client/hmi-client/src/page/data-explorer/components/search-bar.vue

```diff
@@ -296,6 +304,9 @@ i {
 }
 
 .pi.pi-times.clear-search {
+	padding: 0.5rem;
+	border-radius: 1rem;
```

Comment:
Please use `--border-radius` or `--big-border-radius`, or create a new one of `--huge-border-radius: 12px`